leaks.go

```diff
@@ -27,15 +27,15 @@ func start(opts *Options) {
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
 
 	//get current working directory
-	dir,err := os.Getwd()
+	appRoot, err := os.Getwd()
```

Comment:
appRoot is already defined here: https://github.com/zricethezav/gitleaks/blob/master/main.go#L24

Reply:
Ya,changing again.